controller/persistence/appwan_store.go

```diff
@@ -22,11 +22,6 @@ import (
 	"go.etcd.io/bbolt"
 )
 
-const (
```

Comment:
Should this be moved to the /migration package?

Reply:
The appwan store is still referenced by other other stores, to establish the links so moving it would introduce a circular dependency. I'm looking forward to when we can push out an updated version and figure out a time frame for removing all the deprecated pieces.